server/migrations/20170524175710-create-user.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Users', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      email: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      password: {
+        type: Sequelize.TEXT,
+        allowNull: false
+      },
+      profile_id: {
+        type: Sequelize.INTEGER,
+        allowNull: true,
+        onDelete: 'CASCADE',
+        references: {
+          model: 'Profiles',
+          key: 'id'
+        }
+      },
+      is_activate: {
+        type: Sequelize.BOOLEAN,
+        allowNull: false
+      },
+      status_state: {
+        type: Sequelize.STRING,
+        allowNull: false
+      },
+      created_at: {
```

Comment:
do you have something like `timestamps` ? it creates both `created_at` and `updated_at` fields